tubular/edx_api.py

```diff
@@ -48,16 +52,37 @@ def get_api_url(self, path):
         if self.append_slash:
             path += '/'
 
-        return urljoin(f"{self.api_base_url}/", path)
-
-    def _request(self, method, url, **kwargs):
-        response = requests.request(method, url, auth=SuppliedJwtAuth(self._access_token), **kwargs)
+        return urljoin(f'{self.api_base_url}/', path)
 
+    def _request(self, method, url, log_404_as_error=True, **kwargs):
         try:
+            response = requests.request(method, url, auth=SuppliedJwtAuth(self._access_token), **kwargs)
             response.raise_for_status()
         except HTTPError as exc:
-            exc.content = response.content
+            import pdb
```

Comment:
Please remove testing artifacts from the code

Reply:
my bad, removed